src/node_crypto.cc

```diff
@@ -3811,20 +4568,20 @@ void Verify::VerifyFinal(const FunctionCallbackInfo<Value>& args) {
   Verify* verify;
   ASSIGN_OR_RETURN_UNWRAP(&verify, args.Holder());
 
-  char* kbuf = Buffer::Data(args[0]);
-  ssize_t klen = Buffer::Length(args[0]);
+  unsigned int offset = 0;
```

Comment:
```suggestion
  size_t offset = 0;
```
AFAIK `unsigned int` is lint

Reply:
I thought so, too, but `make lint` does not complain about it.

Reply:
It might be a linter bug, anyway IMO `size_t` is a better fit.

Reply:
Yeah. IMO `size_t` is the correct type.
I'm very much against propagating sub-optimal design (I also tend to assume this V8 code would not be accepted now if it was new).

Reply:
Actually it doesn't matter. Go with `int`